doc/classes/SpatialMaterial.xml

```diff
@@ -201,10 +240,13 @@
 			The strength of the normal map's effect.
 		</member>
 		<member name="normal_texture" type="Texture" setter="set_texture" getter="get_texture">
+			Texture used to specify normal at a given pixel. Normal textures only use the red and green channels. Normal read from normal_texture is oriented around the surface normal provided by mesh.
```

Comment:
from `[code]normal_texture[/code]`.

Reply:
to specify *the* normal
provided by *the* mesh (or mesh-provided normal I guess)